README.md

```diff
@@ -1 +1,68 @@
 # cucumber-js-appium-app-browserstack
+
+This repository demonstrates how to run Appium Cucumber JS tests on BrowserStack App Automate.
+
+## Setup
+
+### Requirements
+
+1. Node.js
+
+    - If not installed, install Node.js from [here](https://nodejs.org/en/download/)
+
+    - Ensure you have node & npm installed by running `node -v` & `npm -v`
+
+### Install the dependencies
+
+For Android tests, run the following command in project's base directory :
+
+```
+cd android
+npm install
+```
+Or,
+
+For dependencies for iOS tests, run following command in project's base directory :
+```
+cd ios
+npm install
+```
+## Getting Started
+
+Getting Started with Appium tests in NodeJS mocha on BrowserStack couldn't be easier!
+
+### Run your first test :
+
+Open `Android` or in `ios` folder
+
+- If you have uploaded your app then add the app id to the `browserstack.yml` config file, or you can directly specify the path to your app in the `browserstack.yml` file.
+
+- Run `npm run sample-test`
+
+- You can access the test execution results, and debugging information such as video recording, network logs on [App Automate dashboard](https://app-automate.browserstack.com/dashboard)
+
+---
+
+### **Use Local testing for apps that access resources hosted in development or testing environments :**
+
+Open `Android` or in `ios` folder
+
+- Ensure that `browserstackLocal` capability is set to `true` in the `browserstack.yml` file.
+
+- If you have uploaded your app then add the app id to the `browserstack.yml` config file, or you can directly specify the path to your app in the `browserstack.yml` file.
+
+- Run `npm run sample-local-test`
+
+- You can access the test execution results, and debugging information such as video recording, network logs on [App Automate dashboard](https://app-automate.browserstack.com/dashboard)
+
+## Integration with other NodeJS frameworks
+
+For other NodeJS frameworks samples, refer to following repositories :
+
+- [WebdriverIO](https://github.com/browserstack/webdriverio-appium-app-browserstack)
```

Comment:
Do we want to add other sample repos as well? playwright, codecept..?

Reply:
Not doing as part of this activity.